src/python_routing_v02_diffusive/_restclient/_restclient.py

```diff
@@ -0,0 +1,218 @@
+"""
```

Comment:
Note this is available in [noaa-owp/hydrotools](https://github.com/HydroTools/hydrotools) python package